xrdp/xrdp_wm.c

```diff
@@ -1930,7 +1930,10 @@ callback(intptr_t id, int msg, intptr_t param1, intptr_t param2,
             rv = xrdp_wm_process_channel_data(wm, param1, param2, param3, param4);
             break;
         case 0x5556:
-            rv = xrdp_mm_check_chan(wm->mm);
+            if (!wm->disable_channel_input_check)
```

Comment:
Just a thought, this double-negative expression is a little little bit confusing to me non-native English speaker. 
The current code is acceptable because I always see such double-negative conditions everywhere, every project.